README.md

```diff
@@ -1,2 +1,3 @@
 # CTFtime-Discord-hooks
-![example](https://cdn.discordapp.com/attachments/431375159182622722/500659282593841182/unknown.png)
+![example](https://user-images.githubusercontent.com/50090727/142731112-4c336221-745b-414b-9c30-5719994c9c4d.png)
```

Comment:
please show the full message on the image

Reply:
@MeitarR You can update it after PR :) Might you do it for me?